src/mds/journal.cc

```diff
@@ -1699,6 +1761,10 @@ void ESession::decode(bufferlist::const_iterator &bl)
   } else if (struct_v >= 5) {
     decode(client_metadata, bl);
   }
+  if (struct_v >= 6){
```

Comment:
Update `DECODE_START_LEGACY_COMPAT_LEN(6, 3, 3, bl);`

Reply:
updated it.